Complete stageuser API

dkupka opened the pull request:
https://fedorahosted.org/freeipa/ticket/6623

MartinBasti commented:
LGTM except first commit that shouldn't be here and  `ipalib.x509: Handle missing SAN gracefully` has no ticket in commit message

dkupka commented:
I've removed the first commit, thank you for noticing, I've probably just rebased on branch containing it and then forget to remove it.
There's no ticket for handling missing SAN in ipalib.x509 because i just uncovered newspaper when I replaced the certificates in our test and it was easier to just fix it than reproducing and reporting it.

MartinBasti commented:
Fixed upstream
master:
https://fedorahosted.org/freeipa/changeset/9c0e86530ec693606ca4f69e74a9dfe4118a21aa
https://fedorahosted.org/freeipa/changeset/7e2d185ba09382a815e9b0530aeae3d56f9378d1
https://fedorahosted.org/freeipa/changeset/308c790ee90f00e0bc2c40abf51c30e5250631e9
https://fedorahosted.org/freeipa/changeset/7b68cc5b08c5563535486d72f37b766209791dbf
https://fedorahosted.org/freeipa/changeset/c5c98af99db53b5f9453bf70e9fd4c11e219cf3e
https://fedorahosted.org/freeipa/changeset/9382efde4fbc027dcfb5dc5f22d25296f232e0a6
https://fedorahosted.org/freeipa/changeset/8e139d4b559a6f19d859e078e1940a69d8977fdb
